### PR TITLE
fix: use Keyv instance directly as got request cache

### LINKED ISSUE #5369: Error in log pointing to possible memory leak
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within Z-Wave JS UI (formerly ZwaveJS2MQTT)?

NO, my problem is NOT within Z-Wave JS UI

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

I am not sure if this is important, but will log the issue to see, as I can't find any useful information from googling this:

What causes the bug?
Not sure

What do you observe?
Only that this error is written in the log.

As soon as I do a restart av the Add-on this always arises.

The error that is logged is the following:
(node:157) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 error listeners added to [Keyv]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)

Is this something known that I should only disregard, or is it something you think we should dig deeper into?

### Device information

Manufacturer: 
Model name: 
Node ID in your network: 

### How are you using `node-zwave-js`?

- [X] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker image (latest)
- [ ] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker image (dev)
- [ ] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [ ] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

zwave-js-ui: 8.8.0
zwave-js: 10.4.0
home id: 4110551221
home hex: 0xf50208b5

### Did you change anything?

no

### If yes, what did you change?

_No response_

### Did this work before?

Don't know, this is a new device

### If yes, where did it work?

_No response_

### Attach Driver Logfile

Will attach log if you think this issue is worth digging into. (Am in the middle of a long running firmware update, so dont want to restart and change loglevel now)

### PR DESCRIPTION
**Pull request by AlCalzone**
fixes: #5369

### CODE REVIEW

**Comment by AlCalzone** [comment]
@zwave-js-bot approve

File: packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts [hunk 1]
```diff
@@ -20,7 +21,11 @@ function serviceURL(): string {
 const DOWNLOAD_TIMEOUT = 60000;
 // const MAX_FIRMWARE_SIZE = 10 * 1024 * 1024; // 10MB should be enough for any conceivable Z-Wave chip
 
-const requestCache = new Map();
+// Need to use a Keyv instance directly instead of a map, otherwise each request would
+// create a new Keyv instance, causing event emitter warnings
+// See https://github.com/jaredwray/cacheable-request/pull/101 for details
+const requestCache = new Keyv();
+requestCache.setMaxListeners(10000);
```
Comment (robertsLando) [inline packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts:28]: Question, seems you are not passing requestCache anywhere here?

Comment (AlCalzone) [reply packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts:28]: https://github.com/zwave-js/node-zwave-js/blob/1da050b9e1b69a13b75aa4696a29518640eeaf27/packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts#L102

Comment (robertsLando) [reply packages/zwave-js/src/lib/controller/FirmwareUpdateService.ts:28]: Oh, missed that :)

**Pull request merged**